src/Microsoft.Health.Fhir.Core/Data/OperationDefinition/search-parameter-status.json

```diff
@@ -0,0 +1,32 @@
+{
+    "resourceType": "OperationDefinition",
+    "id": "status",
+    "url": "[base]/OperationDefinition/search-parameter-status",
+    "version": "1.0.0",
+    "name": "Selectable search parameters",
+    "status": "active",
+    "kind": "operation",
+    "description": "Allows for disabling of search parameters that are not being utilized in order to save space in the datastore. This operation is asynchronous as defined in the [FHIR Asynchronous Request Pattern](http://hl7.org/fhir/async.html)",
```

Comment:
nit: this allows customers to either disable or turn it back on as "supported" from disabled state correct? Can we change the description to add something about updating status to supported as well?